src/main/java/org/openpnp/gui/AlphaIcon.java

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
General comment: If you are using code that is from an external source, as some of these files appear to be, can you please include information in the header as to the source of the file and it's licensing terms? For instance, I will often include a link to a StackOverflow page where I found a snippet.